src/win32.ts

```diff
@@ -1,22 +1,10 @@
 import path from 'path';
-import { WrapperError } from 'cross-spawn-windows-exe';
```

Comment:
issue: i think we can fully remove `cross-spawn-windows-exe` from the deps as well